third-party/fedora33/qemu/hoist.bzl

```diff
@@ -0,0 +1,18 @@
+load('//antlir/bzl:oss_shim.bzl', 'buck_genrule')
+
+def hoist(name, out, layer, path, **buck_genrule_kwargs):
```

Comment:
I like this, we may want to have it be generic at some point (since `rpmbuild.bzl` does something similar) but it's good to keep it isolated to `qemu` for now